DigitalLearningSolutions.Web/Views/Frameworks/Developer/_SignpostingResourceCard.cshtml

```diff
@@ -0,0 +1,67 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Frameworks;
+@using DigitalLearningSolutions.Web.Helpers;
+@using DigitalLearningSolutions.Data.Models.External.LearningHubApiClient;
+@model ResourceMetadata
+
+@{
+  var parent = (CompetencyResourceSignpostingViewModel)ViewData["parent"];
+}
+<details class="nhsuk-details nhsuk-expander  nhsuk-u-margin-bottom-3">
+  <summary class="nhsuk-details__summary">
+    <span class="nhsuk-details__summary-text">
+      @SignpostingHelper.DisplayText(Model.Title)
+    </span>
+  </summary>
+  <div class="nhsuk-card__content nhsuk-u-padding-bottom-3 nhsuk-u-padding-top-0 nhsuk-u-margin-1">
+    @if (!String.IsNullOrEmpty(Model.ResourceType))
+    {
+      <div class="nhsuk-grid-row nhsuk-u-margin-left-1 nhsuk-u-clear">
+        <strong class="nhsuk-tag nhsuk-tag--@(SignpostingHelper.DisplayTagColour(Model.ResourceType))">@Model.ResourceType</strong>
+      </div>
+    }
+    <div class="nhsuk-u-margin-left-1 nhsuk-u-margin-top-3 nhsuk-u-margin-bottom-3">
+      @SignpostingHelper.DisplayText(Model.Description)
+    </div>
+    @if (Model.References.Count() > 0)
+    {
+      <div class="nhsuk-u-font-weight-bold nhsuk-u-margin-left-1 nhsuk-u-margin-bottom-2 nhsuk-u-margin-top-2">
+        Catalogues
+      </div>
+      @foreach (var reference in Model.References)
+      {
+        if (reference.Catalogue != null)
+        {
+          <div class="nhsuk-card__content nhsuk-u-margin-1 nhsuk-u-padding-0">
+            <dl class="nhsuk-summary-list">
+              <div class="nhsuk-summary-list__row">
+                <dt class="nhsuk-summary-list__key">
+                  @reference.Catalogue.Name
+                </dt>
+                <dd class="nhsuk-summary-list__value">
+                  <button class="nhsuk-button nhsuk-button--secondary small-edit-button">
+                    Preview
+                  </button>
+                  <button class="nhsuk-button small-edit-button"
+                          asp-action="AddCompetencyLearningResourceSummary"
+                          asp-route-referenceId="@reference.RefId"
+                          asp-route-frameworkId="@parent.FrameworkId"
+                          asp-route-frameworkCompetencyId="@parent.FrameworkCompetencyId"
+                          asp-route-frameworkCompetencyGroupId="@parent.FrameworkCompetencyGroupId"
+                          asp-route-resourceName="@parent.Title"
```

Comment:
This is passing the wrong value into the resourceName parameter. It should be 'asp-route-resourceName="@Model.Title"'. This may be irrelevant if we load the data into the view model for the summary page rather than passing it using parameters here. See previous comment.